.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Please provide detailed information about your issue, thank you!
+
+Version info: 
+- Browser version:
+- Storefront type:
+  - [ ] for .NET Framework
+  - [ ] for .NET Core
+- Storefront version: 
+- Theme version: 
+
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce
+1.
+2.
+3.
```

Comment:
@mvktsk Is this template our issues conventions?

Reply:
@t13ka we don't have issues conventions on paper only templates in repositories